packages/url-parser-universal/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { HttpEndpoint, QueryParameterBag, UrlParser } from "@aws-sdk/types";
+import { Endpoint, QueryParameterBag, UrlParser } from "@aws-sdk/types";
```

Comment:
Is this universal package needed still?

Reply:
Yea. Endpoint is expected to be used even in non-http protocols